source/greentea_test_env.cpp

```diff
@@ -18,9 +18,10 @@
 #include <ctype.h>
 #include <cstdio>
 #include <string.h>
-#include "mbed.h"
 #include "greentea-client/test_env.h"
-
+#include "greentea-client/greentea_metrics.h"
+#include "mbed_trace.h"
+#include "platform/mbed_retarget.h"
```

Comment:
Do we need retarget in this file?

Reply:
This is probably not needed. Shall I do the clean up in a separate PR (which will contain necessary changes build this repository standalone with CMake)?

Reply:
Ah actually this file requires `STDOUT_FILENO` from this header.
```c++
extern "C" int greentea_getc() {
    uint8_t c;
    read(STDOUT_FILENO, &c, 1);
    return c;
}
```
It will be removed in the PR to rework IO.

Reply:
Separate PR is OK. Eventually we'll want to make sure each commit (or at the very least, each PR) can build and pass tests.